cmd/kpod/spec.go

```diff
@@ -400,6 +404,9 @@ func (c *createConfig) GetVolumeMounts() ([]spec.Mount, error) {
 				return nil, errors.Wrapf(err, "relabel failed %q", spliti[0])
 			}
 		}
+		if rootProp == "" {
+			options = append(options, "rprivate")
```

Comment:
I am not sure we should do anything here?  Or is this the docker default?

Reply:
Are these covered in the man pages now?

Reply:
yeah, the docker default is "rprivate" https://docs.docker.com/engine/admin/volumes/bind-mounts/#configure-bind-propagation.
They are covered in the man pages